dissect/target/plugins/filesystem/walkfs.py

```diff
@@ -28,6 +28,8 @@
         ("uint32", "uid"),
         ("uint32", "gid"),
         ("string[]", "fstypes"),
+        ("string[]", "vuuid"),
```

Comment:
```suggestion
        ("string[]", "identifiers"),
```
Maybe an idea to change it to `identifiers` now, as that is what we use in the code.